src/pages/Proposal/index.spec.tsx

```diff
@@ -28,6 +28,7 @@ import VOTE_MODAL_TEST_IDS from './VoteModal/testIds';
 import TEST_IDS from './testIds';
 
 vi.mock('hooks/useVote');
+vi.unmock('hooks/useIsFeatureEnabled');
```

Comment:
How about mocking `useIsFeatureEnabled` directly instead?
This way we're not testing the implementation of the hook.

```typescript
(useIsFeatureEnabled as Vi.Mock).mockImplementation(
      ({ name }: UseIsFeatureEnabled) => name === 'voteProposal',
    );
```